src/graphics/context/webgl.rs

```diff
@@ -274,6 +296,10 @@ impl WebGlContext {
         self.gl_ctx.viewport(0, 0, w as i32, h as i32);
     }
 
+    pub(crate) fn set_framebuffer(&self, fb: Option<&Framebuffer>) {
+        self.gl_ctx.bind_framebuffer(gl::FRAMEBUFFER, fb.map(|fb| &fb.fb));
```

Comment:
This is called by `graphics::set_canvas`, it definitely does something as calling it stops stuff from rendering on the main canvas, and clearing it does restore that.